acrobat/scripts/scripts.js

```diff
@@ -461,20 +461,20 @@ const { ietf } = getLocale(locales);
   }
 
   setConfig({ ...CONFIG, miloLibs });
+
+  loadIms().then(() => {
+    const imsIsReady = new CustomEvent('IMS:Ready');
+    window.dispatchEvent(imsIsReady);
+  });
+
   loadLana({ clientId: 'dxdc', tags: 'DC_Milo' });
 
-  // get event back form dc web and then load area
   await loadArea(document, false);
+
   // Setup Logging
   const { default: lanaLogging } = await import('./dcLana.js');
   lanaLogging();
 
-  // IMS Ready
```

Comment:
Removing this comment, since it's really about loading IMS, not just waiting for the ready event like DC_Hosted, below. Code here speaks for itself, I think.